tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartLocaleGrowth.java

```diff
@@ -355,7 +456,7 @@ private static Map<String, FoundAndTotal> addGrowth(
             }
             // END HACK
 
-            if (showMissing) {
+            if (false && showMissing) {
```

Comment:
dead code like this generally shouldn't be kept unless accompanied by an explanatory comment, preferably a TODO comment with an open ticket number, otherwise it leads to unmaintainable code

Reply:
Agreed, but fixing those kinds of things probably does not belong in this kind of PR which is just merging maint branch changes back to main as is.